app/controllers/concerns/foreman_openscap/hostgroups_controller_overrides.rb

```diff
@@ -0,0 +1,31 @@
+module ForemanOpenscap
+  module HostgroupsControllerOverrides
+    def self.included base
+      base.class_eval do
+        def load_vars_for_ajax
```

Comment:
We shouldn't replace core's `load_vars_for_ajax`, we should override it with `alias_method_chain` instead.
See [foreman_salt override](https://github.com/theforeman/foreman_salt/blob/master/app/controllers/foreman_salt/concerns/hostgroups_controller_extensions.rb#L24) for example.

also in `inherit_parent_attributes`